### ISSUE TITLE
Add ability to disable adding to workspace

### ISSUE THREAD

**Issue by ugcodrr** [streetsidesoftware/vscode-spell-checker#2072]
It would be great if there could be the option to configure what comes up in the quick-fix menu. Ideally, there would be a way to disable things such as adding to the workspace. I would like to disable it because I don't want to worry about adding configuration to other people's projects. As well, I want all the same words ignored no matter the project.

![image](https://user-images.githubusercontent.com/36463298/178177652-edf99f42-f268-4b1c-bec2-da3989300858.png)
Getting rid of the 1st option would be a great enhancement to this extension.

**Comment by gotofritz** [comment]
-1 from me - I actually came here to find a way to ADD this entry to the contextual menu, because my version doesn't show "Add to workspace settings". I don't want to end up with a user settings file filled with hundreds of misspellt words.

**Comment by Jason3S** [comment]
@gotofritz,

Is this related to: #2086

**Comment by gotofritz** [comment]
Yes indeed, same issue, it was an external file not in the workspace.
